packages/react/src/refs.tsx

```diff
@@ -91,13 +91,11 @@ function findAllRefs(stateNode: DataState, dataValues: DataValues): Set<string>
 }
 
 function createRefStateManager(
+  dataValues: DataValues,
```

Comment:
this should work but I personally don't like reassigning arguments. I see them as a `const` so what I would do here is name this argument `initDataValues` then do `let dataValues = initDataValues` in the fn body

Reply:
was going to ask you about this, but you beat me to it. that makes sense!